config/settings.yml

```diff
@@ -54,3 +54,6 @@ metadata_status:
 
 # max. MARC records per OAI-XML file; also max. size of one page in ListRecords
 oai_max_page_size: 500
+
+# base ID of the repository used to generate OAI IDs for records
+oai_repository_id: 'pod.stanford.edu'
```

Comment:
More a thought than a blocking issue, but maybe we should nest these?:

```
oai:
  max_page_size: 500
  repository_id: 'pod.stanford.edu'
```

Reply:
yeah, i was thinking the same — there'll probably be more going forward